### ISSUE TITLE
Indexing notes in "search manuscript"

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/CantusDB#843]
From an email from Debra:

"What I had wanted to check was that you have the search within the "indexing notes" field per source, as here:
https://cantusdatabase.org/searchms/123612

On the left side underneath Office/Mass and Genre, you can search for words that are entered in the indexing notes. I think maybe you have this already (I think I have checked with you before). The option I am using today is searching for a length greater than zero, so that I can pull up any/all notes in one particular source."

on OldCantus this looks like this, for reference 
<img width="954" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/ac5d0aaf-c2bd-4175-8be0-9b130cee619c">
and searching for " " (space) gives 180 results on this particular manuscript.

**Comment by annamorphism** [comment]
addendum:
1. this probably wasn't initially included because it is comparatively new to OldCantus
2. This should only be on "search manuscript". Per Debra "Also, Jan put it in only one place: the "search this manuscript" page that you get to in the RH menu. This is the spot for it, I think, since you would likely only want to search within a source for the notes (for now, anyways - and the notes aren't tidy, so we don’t want to advertise them too much on the main search page)."

**Comment by jacobdgm** [comment]
I'm working on this.

In starting to do so, I realized that the main Chant Search view and the Chant Search MS view, which share a template, have a bunch of copy-pasted and slightly tweaked code, so I'm going to spend some time writing additional tests, and then refactoring these views to make things simpler. (I was starting to work on this for the Search MS view, and in trying to make the clicking-on-column-headers thing work, I realized that with so much duplicated code, I'm really likely to introduce bugs if we don't clean these views up).
